.github/workflows/build_all.yml

```diff
@@ -117,7 +117,7 @@ jobs:
         uses: actions/cache@9b0c1fce7a93df8e3bb8926b0d6e9d89e92f20a7 # pin@v3.0.11
         with:
           path: ~/.vagrant.d/boxes/magmacore-VAGRANTSLASH-magma_dev
-          key: vagrant-box-magma-dev-v1.2.20221012
+          key: vagrant-box-magma-dev-v1.2.20221230
```

Comment:
Would it make sense to switch to `vagrant-box-magma-dev-v1.3.20221230` for the key? Same for the others.

Reply:
I was not part of the `v1.1` -> `v1.2` discussion, so I can't comment on any real convention. Previously, we just adapted the timestamp.

Reply:
This is perhaps a point to be raised in a larger round then. I would suggest to move to a common version for the new VMs.

Reply:
The version was updated to 1.3 in the cloud and the PR.